bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py

```diff
@@ -121,6 +122,14 @@ def get_matched_id(subfields):
     return None
 
 
+def salvage_deleted_record_from_history(recid):
+    return create_record(decompress(run_sql("SELECT marcxml FROM hstRECORD WHERE id_bibrec=%s ORDER BY job_date DESC LIMIT 1", (recid, ))[0][0]))[0]
```

Comment:
Yeah, I have the same comment: as far as I understand, `bibfmt` is the ultimate source of truth of records, and any other derived table could be losing data because of bugs in the code that derives that table. Or am I wrong?